packages/amplify-category-geo/src/service-utils/mapUtils.ts

```diff
@@ -28,7 +34,8 @@ export const createMapResource = async (context: $TSContext, parameters: MapPara
   const mapStack = new MapStack(new App(), 'MapStack', { ...parameters, ...templateMappings, authResourceName });
   generateTemplateFile(mapStack, parameters.name);
   saveCFNParameters(parameters);
-  stateManager.setResourceInputsJson(pathManager.findProjectRoot(), category, parameters.name, { groupPermissions: parameters.groupPermissions });
+  stateManager.setResourceInputsJson(pathManager.findProjectRoot(),
+    category, parameters.name, { groupPermissions: parameters.groupPermissions });
```

Comment:
nit: formatting and pathManager.findProjectRoot() gets called anyways if undefined is passed into the first arrgument

```suggestion
  stateManager.setResourceInputsJson(
    undefined,
    category,
    parameters.name,
    { groupPermissions: parameters.groupPermissions },
  );
```